bindings/java/README.md

```diff
@@ -1,8 +1,18 @@
+# OpenDAL Java Bindings
+
+## Build and test
```

Comment:
Add basic build instruction.

Reply:
https://github.com/apache/incubator-opendal/blob/main/.github/workflows/bindings_java.yml

May we update here together?

Reply:
It may be helpful to add some content related to the environment.

Reply:
You're right. I'll pay attention to my wording. 
We can improve the doc later.